### ISSUE TITLE
Upgrade from 1.4.1 to 1.4.6 failed

### ISSUE THREAD

**Issue by MrMontesa** [zikula/core#3492]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.6
| PHP Version       | 5.6.30

### Expected behavior
Update should not end up in blank page

### Actual behavior
Started an update on one of my pages, all went fine, upgrade wizard passsed. After update on redirecting to /login page I got an error 500. On subsequent reloads the index page remained white and dos not load anything.
Thus I enabled debug, and switched to env = dev. Same result. Page does not load at all.
dev.log shows tons of the following messages:
```
[2017-03-21 16:39:05] php.INFO: ServiceManager is deprecated, please use Symfony container instead. {"type":16384,"file":"/data/web/1/000/027/915/75642/htdocs/sebsys/lib/legacy/Zikula/ServiceManager.php","line":121,"level":28928} []
[2017-03-21 16:39:05] php.INFO: AbstractRequest is deprecated, please use Symfony request instead. {"type":16384,"file":"/data/web/1/000/027/915/75642/htdocs/sebsys/lib/legacy/Zikula/Request/
```
The only thing I forgot to do before updating, was removing News module from start app. I also do have some disabled or inconsistent modules in the tree.
Any idea what could have caused this issue?
### Steps to reproduce
Not sure as this might be related to my setup.

**Comment by Guite** [comment]
The log entries are unrelated.

Do you have the German locales installed?
If yes, ensure that in `app/Resources/translations/routes.de.po` the login route is translated like this:

```
msgid "zikulausersmodule_access_login"
msgstr "/login"
```

**Comment by MrMontesa** [comment]
No german translation installed at all.

**Comment by craigh** [comment]
I may have fixed this in 1.5 can you test it?

**Comment by craigh** [comment]
@MrMontesa

**Comment by craigh** [comment]
closed by c635bf83d4c65765408c6865aef6257602d59374

reopen if issue persists.
